### PR TITLE
updated steps for Intune

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/8706

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @nam31, for your information

Thank you

**Comment by illfated** [comment]
(Note to self: this page really needs a MarkDown code block fence format _cleanup_. #MD-cleanup)

File: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-cert.md [hunk 1]
```diff
@@ -639,47 +636,42 @@ Sign-in a workstation with access equivalent to a _domain user_.
...
-20. Click **Next**.
-21. Click **Next** two more times to skip the **Scope tags** and **Assignments** steps of the wizard and click **Create**.
+10. Select **Enroll to Windows Hello for Business, otherwise fail (Windows 10 and later)** from the **Key storage provider (KSP)** list.
+11. Next to **Subject name format**, type **CN={{OnPrem_Distinguished_Name}}** to make the on-premises distinguished name the subject of the issued certificate.
+12. Specify **User Principal Name (UPN)** as a **Subject Alternative Name** parameter. Set its value as {{UserPrincipalName}}.
+13. Refer to the "Configure Certificate Templates on NDES" task for how you configured the **AADJ WHFB Authentication** certificate template in the registry. Select the appropriate combination of key usages from the **Key Usages** list that map to configured NDES template in the registry. In this example, the **AADJ WHFB Authentication** certificate template was added to the **SignatureTemplate** registry value name. The **Key usage** that maps to that registry value name is **Digital Signature**.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-cert.md:655]:
```suggestion
13. Refer to the "Configure Certificate Templates on NDES" task for how you configured the **AADJ WHFB Authentication** certificate template in the registry. Select the appropriate combination of key usages from the **Key Usages** list that map to the configured NDES template in the registry. In this example, the **AADJ WHFB Authentication** certificate template was added to the **SignatureTemplate** registry value name. The **Key usage** that maps to that registry value name is **Digital Signature**.
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-cert.md [hunk 2]
```diff
@@ -639,47 +636,42 @@ Sign-in a workstation with access equivalent to a _domain user_.
...
-![WHFB Scep Profile landing](images/aadjcert/intunewhfbscepprofile-04.png)
-5. Click **Assignments**.
-6. In the **Assignments** pane, Click **Include**.  Select **Selected Groups** from the **Assign to** list.  Click **Select groups to include**.
-![WHFB SCEP Profile Assignment](images/aadjcert/intunewhfbscepprofileassignment.png)
-7. Select the **AADJ WHFB Certificate Users** group. Click **Select**.
-8. Click **Save**.
+1. Sign-in to the [Microsoft Endpoint Manager admin center](https://endpoint.microsoft.com/).
+2. Select **Devices**, and then click **Configuration Profiles**.
+3. Click **WHFB Certificate Enrollment**.
+4. Select **Properties**, click **Edit** next to the **Assignments** section.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-cert.md:670]:
```suggestion
4. Select **Properties** and click **Edit** next to the **Assignments** section.
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-cert.md [hunk 3]
```diff
@@ -639,47 +636,42 @@ Sign-in a workstation with access equivalent to a _domain user_.
...
+15. Under **Extended key usage**, type **Smart Card Logon** under **Name**. Type **1.3.6.1.4.1.311.20.2.2** under **Object identifier**. Click **Add**.
+16. Type a percentage (without the percent sign) next to **Renewal Threshold** to determine when the certificate should attempt to renew. The recommended value is **20**.
+    ![WHFB SCEP certificate Profile EKUs](images/aadjcert/profile03.png)
+17. Under **SCEP Server URLs**, type the fully qualified external name of the Azure AD Application proxy you configured. Append to the name **/certsrv/mscep/mscep.dll**. For example, https://ndes-mtephendemo.msappproxy.net/certsrv/mscep/mscep.dll. Click **Add**. Repeat this step for each additional NDES Azure AD Application Proxy you configured to issue Windows Hello for Business certificates. Microsoft Intune round-robin load balances requests among the URLs listed in the SCEP certificate profile.
+18. Click **Next**.
+19. Click **Next** several times to skip the **Scope tags**, **Assignments** and **Applicability Rules** steps of the wizard and click **Create**.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-cert.md:662]:
```suggestion
19. Click **Next** several times to skip the **Scope tags**, **Assignments**, and **Applicability Rules** steps of the wizard and click **Create**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - three minor changes.

**Review by mapalko — APPROVED** [review]

**Review by illfated — APPROVED** [review]

**Comment by illfated** [comment]
(ref. issue ticket #8548 **Error 500 if no Value for UPN is given**)

**Pull request merged**